docs/source/quickstart.mdx

```diff
@@ -1,211 +1,285 @@
+# Quickstart
+
+This quickstart is intended for developers ready to dive into the code and see an end-to-end example of how to integrate 🤗 Datasets into their model training workflow. If you are a beginner or a non-developer, we recommend starting with our [tutorials](./tutorial), where you will get a more thorough introduction.
+
+Each dataset is unique, and depending on the task, some datasets may require additional steps to prepare the data. But you can always use 🤗 Datasets tools to load and process a dataset. Choose the type of dataset you want to work with, and let's get started!
+
+<div class="mt-4">
+   <div class="w-full flex flex-col space-y-4 md:space-y-0 md:grid md:grid-cols-3 md:gap-y-4 md:gap-x-5">
+      <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#audio"
+         ><div class="w-full text-center bg-gradient-to-r from-violet-300 via-sky-400 to-green-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Audio</div>
+         <p class="text-gray-700">Resample an audio dataset and get it ready for a model to classify what type of banking issue a speaker is calling about.</p>
+      </a>
+      <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#vision"
+         ><div class="w-full text-center bg-gradient-to-r from-pink-400 via-purple-400 to-blue-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Vision</div>
+         <p class="text-gray-700">Apply data augmentation to an image dataset and get it ready for a model to diagnose disease in bean plants.</p>
+      </a>
+      <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#nlp"
+         ><div class="w-full text-center bg-gradient-to-r from-orange-300 via-red-400 to-violet-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">NLP</div>
+         <p class="text-gray-700">Tokenize a dataset and train a model to determine whether a pair of sentences have the same meaning.</p>
+      </a>
+   </div>
+</div>
 
 <Tip>
 
-For more detailed information on loading and processing a dataset, take a look at [Chapter 3](https://huggingface.co/course/chapter3/1?fw=pt) of the Hugging Face course! It covers additional important topics like dynamic padding, and fine-tuning with the Trainer API.
+For more detailed information about the 🤗 Datasets library, check out [Chapter 5](https://huggingface.co/course/chapter5/1?fw=pt) of the Hugging Face course! It covers other important topics such as loading remote or local datasets, tools for cleaning up a dataset and creating your own dataset.
```

Comment:
Can we use a different wording here? As I understand, this would mean that our tutorial offers more info about the lib than the lib docs, which doesn't make sense 😄.